src/workflows-app/SubmissionConfig.js

```diff
@@ -222,9 +226,10 @@ export const BaseSubmissionConfig = (
         });
       } else if (cbasUrlRoot) {
         loadRunSet(cbasUrlRoot).then((runSet) => {
-          setRunSetRecordType(runSet.record_type);
```

Comment:
I'm probably just missing something, but could you explain why this is getting removed?

Reply:
Good callout, I should have left a comment explaining this. 2 main reasons - 1) In the original component, we were tracking `selectedRecordType` and `runSetRecordType`, which were consistently being set to the same value (in this case, `selectedRecordType` was set by `loadRunSet`, and `runSetRecordType` was set immediately afterwards. Since they were being used interchangeably it seemed to make sense to only track one of them. 2) The reason I noticed this in the first place is because on line 227/231 when we load in the WDS data we pass in `runSetRecordType`, which despite being set on line 225 will still be undefined on 227 due to the way that React handles state. By simply using the `runSet.record_type` returned from `loadRunSet` we'll now use the most up to date record type, allowing for the expected "auto-load previously selected data table" behavior which was buggy at first.

Reply:
I may have incorrectly conflated these variables... doing some digging now.